go-controller/pkg/node/node.go

```diff
@@ -478,6 +494,184 @@ func (n *OvnNode) Start(wg *sync.WaitGroup) error {
 	return err
 }
 
+func (n *OvnNode) WatchCRD() {
+	n.watchFactory.AddCRDHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc: func(obj interface{}) {
+			crd := obj.(*apiextension.CustomResourceDefinition)
+			klog.Infof("Adding CRD %s to cluster", crd.Name)
+			// currrently the only reason we watch crds is to determine if the watchers for egressfirewall
+			// should be running
+			if n.egressfirewallEnabled {
```

Comment:
Same comments as `WatchCRD` for ovnkube-master + maybe, this should be a util func somewhere so that we can avoid the duplication of code?